app/views/groups/index.html.haml

```diff
@@ -1,9 +1,26 @@
+.groups-page-description
+  %h1= t("pages.index.groups")
+  %p= t("pages.index.groups_info")
 
-%h1= t("pages.index.groups")
+.groups-showcase-container
+  - @group_types.each do |t|
+    .samf-container.plain.p-3.m-2.mt-3
+      .flex-row.wrap.center
+        .group-type-headline=t
+        - t.groups.each do |g|
```

Comment:
If number of groups with type is empty, do not render container.
Must count number of groups without pages